server.go

```diff
@@ -634,6 +647,30 @@ func (srv *notebookServer) newNotebookHandler(w http.ResponseWriter, r *http.Req
 	handlerPath := path.Join("/book", tmpnb.hash) + "/"
 	log.Printf("handler: %s", handlerPath)
 	handler := func(w http.ResponseWriter, r *http.Request) {
+		//BUG If a user provides a fictitious cookie that could
+		//represent another user, they could theoretically gain access
+		//to a notebook which does not belong to them.
+
+		//Read the cookie for session information and compare the
+		//email to the email provided by the tmpnb. If they match,
+		//allow access, else redirect them to /list
+		e := ""
```

Comment:
See above, also maybe `email` would be better, to be consistent.

Reply:
email is already a variable in scope from the lines above. I just declared another var for simplicity